AutomatedTesting/Gem/PythonTests/editor/EditorScripts/ComponentCRUD_Add_Delete_Components.py

```diff
@@ -125,7 +125,8 @@ async def add_component(component_name):
         success = await pyside_utils.wait_for_condition(lambda: not hydra.has_components(entity_id, ['Mesh']), 5.0)
         Report.result(Tests.mesh_component_deleted, success)
 
-        # 7) Undo deletion of component
+        # 7) Undo deletion of component after waiting for the deletion to register
+        general.idle_wait(0.5)
```

Comment:
Does the `wait_for_propagation` work here or no?

Reply:
It does not, sadly. I don't think it's waiting on prefab propagation here, so required a longer wait than 1 frame.

Reply:
Was actually able to update this to wait_for_propagation. A previous part of the test that was waiting for an entity inspector refresh was what actually needed more time.